extensions/ql-vscode/src/model-editor/model-evaluator.ts

```diff
@@ -105,6 +106,11 @@ export class ModelEvaluator extends DisposableObject {
     }
   }
 
+  public async openModelAlertsView() {
+    const view = new ModelAlertsView(this.app);
```

Comment:
Long term we'll need some logic to track to open views and avoid you opening two identical views. But that could be in a followup, so long as we don't forget it.